Logos/atlassian.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="75" height="75" fill="none" xmlns:v="https://vecta.io/nano"><g fill="#0052cc"><path d="M4.382 31.145c-.562.864-.889 1.86-.95 2.889s.147 2.057.603 2.981l1.514 3.027a.27.27 0 0 0 .099.109c.043.026.092.04.142.04h3.139a.27.27 0 0 0 .241-.39l-4.33-8.658c-.021-.043-.054-.079-.094-.105S4.658 31 4.611 31s-.095.014-.135.039-.073.062-.094.105zm-1.654 4.1a.26.26 0 0 0-.105-.082c-.041-.018-.087-.024-.131-.02a.26.26 0 0 0-.124.047c-.037.026-.066.061-.085.102L.028 39.801a.27.27 0 0 0 .11.356.27.27 0 0 0 .131.034h3.14a.26.26 0 0 0 .241-.149c.677-1.399.267-3.527-.922-4.797zm41.016 1.908c-2.026-.391-2.543-1.278-2.543-2.394 0-1.393 1.013-2.244 2.969-2.244a5.76 5.76 0 0 1 2.497.518v1.738c-.817-.368-1.68-.633-2.555-.633-.725 0-1.116.184-1.116.621 0 .345.265.587 1.358.829 1.979.426 2.682 1.047 2.682 2.371 0 1.403-.84 2.348-2.958 2.348-1.496 0-2.302-.334-2.912-.633v-1.818a6.23 6.23 0 0 0 2.935.794c.851 0 1.105-.253 1.105-.644 0-.403-.253-.598-1.462-.852zm24.662-4.523v7.561h1.611v-5.766l.679 1.53 2.279 4.235H75v-7.56h-1.611v4.88l-.61-1.416-1.83-3.464h-2.543zm-12.009 0h1.761v7.561h-1.761V32.63zm-2.032 5.329c0-1.324-.702-1.945-2.681-2.371-1.093-.242-1.358-.483-1.358-.829 0-.437.391-.621 1.116-.621.875 0 1.738.265 2.555.633v-1.738c-.784-.357-1.637-.534-2.498-.518-1.956 0-2.969.852-2.969 2.244 0 1.116.518 2.002 2.543 2.394 1.209.254 1.462.449 1.462.852 0 .391-.254.644-1.105.644a6.23 6.23 0 0 1-2.935-.794v1.818c.61.299 1.415.633 2.912.633 2.118 0 2.958-.944 2.958-2.348zm-26.843 2.232V32.63h1.772v5.927h2.417l-.57 1.634h-3.619zm-7.15-7.561v1.634h1.956v5.927H24.1v-5.927h2.095V32.63h-5.823z"/><g fill-rule="evenodd"><use xlink:href="#B"/><path d="M37.381 32.63h-2.322l-2.637 7.561h2.014l.374-1.273c.92.27 1.899.27 2.82 0l.374 1.273h2.014l-2.636-7.561zm-1.161 4.924c-.328 0-.654-.047-.969-.139l.969-3.3.969 3.301a3.44 3.44 0 0 1-.969.138z"/><use xlink:href="#B" x="46.702"/></g></g><defs ><path id="B" d="M15.479 32.63h2.323l2.636 7.561h-2.014l-.374-1.273c-.921.27-1.899.27-2.82 0l-.374 1.273h-2.014l2.637-7.561zm.192 4.785c.315.093.641.139.969.139a3.44 3.44 0 0 0 .969-.138l-.969-3.301-.969 3.3z"/></defs></svg>
```

Comment:
Can you please correct the issues with Logo? 
Ensure raw file of logo does not have xmlns:xlink. 
id should have Guid. If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets 

Refer https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md for logo guidelines.

Reply:
Done, pls check new logo